cli/cmd/ui.go

```diff
@@ -63,9 +70,29 @@ var uiCmd = &cobra.Command{
 			fmt.Printf("Error starting UI: %v\n", err)
 			os.Exit(1)
 		}
+
 	},
 }
 
+func checkOdigosInstallation(cmd *cobra.Command) bool {
+	client, err := kube.CreateClient(cmd)
+	if err != nil {
+		kube.PrintClientErrorAndExit(err)
+		return false
+	}
+	ctx := cmd.Context()
+
+	// check if odigos is installed
+	_, err = resources.GetOdigosNamespace(client, ctx)
+	if err == nil {
+		return true
+	} else if !resources.IsErrNoOdigosNamespaceFound(err) {
+		fmt.Printf("\033[31mERROR\033[0m Cannot install/start UI. Failed to check if Odigos is already installed: %s\n", err)
+		return false
+	}
+	return true
```

Comment:
shouldn't we return `false` in this case?
this line will hit when the namespace is not found (which means odigos is not installed).

If this function returns `true` and the caller above checks for `if !checkOdigosInstallation(cmd)`, it means we will not `os.Exit` and will attempt to run the UI